packages/slice-machine/src/modules/useSliceMachineActions.ts

```diff
@@ -402,14 +402,12 @@ const useSliceMachineActions = () => {
   const generateSliceCustomScreenshot = (
```

Comment:
should this file be in the hooks directory?

Reply:
I think it was created there to be close to the redux logic as we modify it only when we modify redux actions.
If you want to move it I won't vote against it but I think it would be painful to find it each time we have to modify it. 👍

So do what you want, let us know.